src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineMultiline.java

```diff
@@ -161,22 +161,22 @@ private void issue2211pass2() {
 
     /**
      * Multiple statements within try-with-resource on next line after try is illegal.
-     * @see <a href="https://github.com/checkstyle/checkstyle/issues/2211">OneStatementPerLine: false match with try-with-resources</a>
+     * @see <a href="https://github.com/checkstyle/checkstyle/issues/2211">false match</a>
      */
     private void issue2211fail() {
         try(
-                AutoCloseable i = new java.io.StringReader("");AutoCloseable k = new java.io.StringReader("");
+      AutoCloseable i = new java.io.StringReader("");AutoCloseable k = new java.io.StringReader("");
         ) {
         } catch (Exception e1) {
         }
     }
 
     /**
-     * Multiple statements within try-with-resource on a same line as try is illegal.  Added per PR comment:
-     * @see <a href="https://github.com/checkstyle/checkstyle/pull/2750#issuecomment-166032327">Please add UT cases</a>
+     * Multiple statements within try-with-resource on a same line as try is illegal. PR comment:
+     * @see <a href="https://github.com/checkstyle/checkstyle/pull/2750#issuecomment-166032327"/>
      */
     private void issue2211fail2() {
-        try( AutoCloseable i = new java.io.StringReader("");AutoCloseable k = new java.io.StringReader("");) {
+        try( AutoCloseable i = new StringReader("");AutoCloseable k = new StringReader("");) {
```

Comment:
I see some other test cases where we kept the full classpath, so this should be ok.